services/workflows-service/src/workflow/schemas/zod-schemas.ts

```diff
@@ -58,6 +58,7 @@ export const ConfigSchema = z
     kybOnExitAction: z.enum(['send-event', 'redirect-to-customer-portal']).optional(),
     reportConfig: z.record(z.string(), z.unknown()).optional(),
     theme: WorkflowDefinitionConfigThemeSchema.optional(),
+    ubosOngoingMonitoring: z.boolean().optional(),
```

Comment:
A bit more clarity that its a boolean and a hope for consistency/conventions. Again, not critical.
```suggestion
    isUbosOngoingMonitoring: z.boolean().optional(),
```

Reply:
Thanks